### DISCUSSION TITLE
How to enable swipe to adjust brightness with touchpad?

### DISCUSSION THREAD

**Discussion by DavidVazquezBK** [emoacht/Monitorian#189] [Q&A]
Regarding last update, how do I enable this feature?
I already updated to 2.19 but I can't seem to find a way to to it, I do have a precision  touchpad.
Thanks!

**Comment by emoacht** [comment]
Thanks for trying that feature.
It is really simple.

1. Show this app's main window. If you are not sure that the window is in foreground, click the window.
2. Place two fingers on the touchpad and swipe right or left. You can move the cursor anytime by one finger gesture.

If this app still does not respond, let me know. I am preparing an app to look into the issue.
